purchase_order_line_price_history_discount/wizards/purchase_order_line_price_history.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class PurchaseOrderLinePriceHistoryline(models.TransientModel):
+    _inherit = "purchase.order.line.price.history.line"
+
+    discount = fields.Float(related="purchase_order_line_id.discount")
+
+    def _prepare_purchase_order_line_vals(self):
+        vals = super()._prepare_purchase_order_line_vals()
```

Comment:
```suggestion
    def _prepare_purchase_order_line_vals(self):
        self.ensure_one()
        vals = super()._prepare_purchase_order_line_vals()
```

Reply:
I think in this case it isn't necessary. When the method is called from super(), `self.ensure_one()` will be executed.
https://github.com/OCA/purchase-workflow/pull/953/files#diff-b595047d950f62cc98957ba740167de9R110

Reply:
I agree